### DISCUSSION TITLE
Just wanted to understand what difference kernel su makes when magisk already exist and is working fine too

### DISCUSSION THREAD

**Discussion by Greatz08** [tiann/KernelSU#1529] [General]
In  much simpler way can anyone please explain me the the reason for the birth of this project and what all issues it solves which magisk cant solve due to the way it implements its thing. How much safer it is ,Can it support magisk type modules easily,does it prevent root detection naturally from banking and other advanced apps and games without any external module.These were some of my questions rest you can tell more about it in detail if possible.At last i would like to know if there are  any major disadvantage/disadvantages over magisk solution

**Comment by tiann** [comment]
read the docs first
